src/component/Testimonials.jsx

```diff
@@ -0,0 +1,65 @@
+import { Component } from "react";
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+import { Carousel } from "react-responsive-carousel";
+import "../styles/testimonials.css";
+
+export default class Testimonials extends Component {
+  render() {
+    return (
+      <Carousel
+        showArrows={true}
+        infiniteLoop={true}
+        showThumbs={false}
+        showStatus={false}
+        autoPlay={true}
+        interval={6100}
+      >
+        <div>
```

Comment:
It looks like the div blocks here follow a repeated pattern with only the content changing. You could define a component like `ProfileCard` that takes `imageSrc`, `name`, `role`, ... as props.